src/sentry/api/endpoints/group_details.py

```diff
@@ -336,26 +334,18 @@ def put(self, request, group):
                     )
                 ),
             )
-            metrics.incr(
-                "group.update.http_response",
-                sample_rate=1.0,
-                tags={"status": response.status_code, "detail": "group_details:put:response"},
-            )
             return Response(serialized, status=response.status_code)
         except client.ApiError as e:
+            logging.error(
+                "group_details:put client.ApiError",
+                exc_info=True,
+            )
```

Comment:
I should be able to now include this in a discover query to get numbers much closer to the SLO with 400's included, as this, *I believe* is a huge bulk of them.